service/src/main/java/org/cbioportal/service/impl/CacheStatisticsServiceImpl.java

```diff
@@ -30,44 +30,12 @@ protected void checkIfCacheStatisticsEndpointEnabled() {
 
     @Override
     public List<String> getKeyCountsPerClass(String cacheName) throws CacheNotFoundException {
-        checkIfCacheStatisticsEndpointEnabled();
-        Cache<String, Object> cache = cacheManager.getCache(cacheName);
-        if (cache == null) {
-            throw new CacheNotFoundException(cacheName);
-        }
-        Map<String, Integer> classToKeyCount = new HashMap<String, Integer>();
-        Iterator<Cache.Entry<String, Object>> iterator = cache.iterator();
-        while (iterator.hasNext()) {
-            Cache.Entry<String, Object> entry = iterator.next();
-            String cacheKey = entry.getKey();
-            String className = cacheKey.split("_")[0];
-            int keyCount = classToKeyCount.containsKey(className) ? classToKeyCount.get(className) : 0;
-            classToKeyCount.put(className, keyCount + 1);
-        }
-        List<String> keyCountsPerClass = new ArrayList<String>();
-        for (Map.Entry<String, Integer> entry : classToKeyCount.entrySet()) {
-            keyCountsPerClass.add(entry.getKey().toString() + ": " + entry.getValue().toString() + " keys");
-        }
-        return keyCountsPerClass;
+        return new ArrayList<>();
```

Comment:
Are we returning an empty list to disable some default behavior?

Reply:
I knew I deleted a bunch of code somewhere. Good catch, this has to be reimplemented.